GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerTestCase/ModifiedPathsTests.cs

```diff
@@ -172,15 +171,22 @@ public static object[] Runners
             {
                 get
                 {
-                    if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                    List<object[]> hardLinkRunners = new List<object[]>();
+                    foreach (object[] runner in FileSystemRunner.Runners.ToList())
                     {
-                        return new[]
+                        FileSystemRunner fileSystem = runner.ToList().First() as FileSystemRunner;

```

Comment:
Nit pick: you're using `as` but not checking for null. I would just do a normal cast here so we end up with a more appropriate exception if someone ever inserts in invalid type into the list.

Reply:
> I would just do a normal cast here so we end up with a more appropriate exception if someone ever inserts in invalid type into the list.

Sounds good I'll fix this up